docs/src/main/asciidoc/spring-di.adoc

```diff
@@ -319,6 +323,14 @@ The following table shows how Spring DI annotations can be converted to CDI and
 |@Scope
 |
 |Doesn't have a one-to-one mapping to a CDI annotation. Depending on the value of @Scope, one of the @Singleton, @ApplicationScoped, @SessionScoped, @RequestScoped, @Dependent could be used
+
+|@ComponentScan
+|
+|Doesn't have a one-to-one mapping to a CDI annotation. Quarkus uses `annotated` mode for discovering beans.
```

Comment:
I would change to: `Is not  used in Quarkus because Quarkus does all classpath scanning at build time`